### PR TITLE
fix: 19 digit user IDs should work.

### PR DESCRIPTION
**Pull request by DEV-DIBSTER**
Untested changes. Should be verified by someone before merge.

### CODE REVIEW

File: bot/discord/commands/staff/premium.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ exports.run = async(client, message, args) => {
         client.channels.cache.get('898041841939783732').send('Thanks, <@' + userid + '> for donating $' + parser.format(amount))
     }
 
-    let userid = message.guild.members.cache.get(args[2].match(/[0-9]{18}/).length == 0 ? args[2] : args[2].match(/[0-9]{18}/)[0]);
+    let userid = message.guild.members.cache.get(args[2].match(/[0-9]{18,19}/).length == 0 ? args[2] : args[2].match(/[0-9]{18,19}/)[0]);
```
Comment (WilliamDavidHarrison) [inline bot/discord/commands/staff/premium.js:17]:
```suggestion
    let userid = message.guild.members.cache.get(args[2].match(/[0-9]{17,19}/).length == 0 ? args[2] : args[2].match(/[0-9]{17,19}/)[0]);
```

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]
There are some accounts with 17-digit IDs

File: bot/discord/commands/staff/premium.js (hunk 1 above)
Comment (Sliden101) [reply bot/discord/commands/staff/premium.js:17]: already merged once broke the entire command

Comment (WilliamDavidHarrison) [reply bot/discord/commands/staff/premium.js:17]: Oh

**Pull request merged**